src/client/app/components/CompareControlsComponent.tsx

```diff
@@ -49,12 +51,6 @@ export default function CompareControlsComponent() {
 	);
 }
 
-const divTopBottomPadding: React.CSSProperties = {
-	paddingTop: '0px',
-	paddingBottom: '15px'
-};
 
```

Comment:
There are a number of files with extra blank lines. I'm going to stop marking them but please fix them all.